server/models/Submission.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+
+const submissionSchema = new Schema({
+    contest_id: {
```

Comment:
Just a suggestion, as MongoDB speaks JavaScript, try to utilize JS naming conventions of camelCase while defining field names :)

Reply:
thank you for the suggestion, and I changed all the naming to camelCase